components/forms/text-areas/text-area.scss

```diff
@@ -1,11 +1,11 @@
 .msds-form {
   &__input-container {
     &-text-area {
+      font-family: 'Open Sans', sans-serif;
```

Comment:
could be a global variable with the family font? I guess is almost always the same